azure-mgmt-cognitiveservices/azure/mgmt/cognitiveservices/models/cognitive_services_account_create_parameters.py

```diff
@@ -47,7 +49,7 @@ class CognitiveServicesAccountCreateParameters(Model):
 
     _attribute_map = {
         'sku': {'key': 'sku', 'type': 'Sku'},
-        'kind': {'key': 'kind', 'type': 'Kind'},
+        'kind': {'key': 'kind', 'type': 'str'},
```

Comment:
@annatisch I think we lost something here. Type is still the same, but Autorest 1.0 is using "str" and not Kind. Thoughts?